cirq/google/optimizers/optimize_for_xmon.py

```diff
@@ -14,72 +14,26 @@
 """A combination of several optimizations targeting XmonDevice."""
 from typing import Callable, cast, List, Optional, TYPE_CHECKING
 
-from cirq import circuits, devices, optimizers
-from cirq.google.optimizers import convert_to_xmon_gates
+from cirq import devices
+from cirq._compat import deprecated
+from cirq.google.optimizers import optimized_for_gmon
 
 if TYPE_CHECKING:
     import cirq
 
-_TOLERANCE = 1e-5
-
-
-def _merge_rots(c: 'cirq.Circuit'):
-    return optimizers.merge_single_qubit_gates_into_phased_x_z(c, _TOLERANCE)
-
-
-_OPTIMIZERS: List[Callable[['cirq.Circuit'], None]] = [
-    convert_to_xmon_gates.ConvertToXmonGates().optimize_circuit,
-    optimizers.MergeInteractions(tolerance=_TOLERANCE,
-                                 allow_partial_czs=False).optimize_circuit,
-    _merge_rots,
-    optimizers.EjectPhasedPaulis(tolerance=_TOLERANCE).optimize_circuit,
-    optimizers.EjectZ(tolerance=_TOLERANCE).optimize_circuit,
-    optimizers.DropNegligible(tolerance=_TOLERANCE).optimize_circuit,
-]
-
-_OPTIMIZERS_PART_CZ: List[Callable[['cirq.Circuit'], None]] = [
-    convert_to_xmon_gates.ConvertToXmonGates().optimize_circuit,
-    optimizers.MergeInteractions(tolerance=_TOLERANCE,
-                                 allow_partial_czs=True).optimize_circuit,
-    _merge_rots,
-    optimizers.EjectPhasedPaulis(tolerance=_TOLERANCE).optimize_circuit,
-    optimizers.EjectZ(tolerance=_TOLERANCE).optimize_circuit,
-    optimizers.DropNegligible(tolerance=_TOLERANCE).optimize_circuit,
-]
-
 
+@deprecated(deadline='v0.7.0',
+            fix='Use optimized_for_gmon with '
```

Comment:
This method does something different than optimized_for_gmon, so it's not really appropriate to just substitute one in.

Reply:
Removed deprecation warning, but it still depends on the underlying function.

Also changed the name to optimize_for_sycamore.